lib/routes/receive_payment/ln_address/receive_lightning_address_page.dart

```diff
@@ -11,7 +11,7 @@ import 'package:misty_breez/widgets/widgets.dart';
 /// Page that displays the user's Lightning Address for receiving payments.
 class ReceiveLightningAddressPage extends StatefulWidget {
   static const String routeName = '/lightning_address';
-  static const PaymentMethod paymentMethod = PaymentMethod.bolt11Invoice;
+  static const PaymentMethod paymentMethod = PaymentMethod.lightning;
```

Comment:
The `lightning` payment method is deprecated and should be replaced with `bolt11Invoice`. Is this an issue for tracking?

Reply:
It's not affecting other UIs although the current payment method logic on `ReceivePaymentPage` could be simplified. [Removed unused payment method from receive pages](https://github.com/breez/misty-breez/pull/533/commits/a142c9d7c175b476482fe88523a4d4c1e3834f98)

I also [renamed paymentMethod to paymentLabel on DestinationWidget & it's children](https://github.com/breez/misty-breez/pull/533/commits/1b61321940535611f547b01bb72eeee75041d8df) for clarity.